src/Components/Checkbox.tsx

```diff
@@ -56,16 +59,15 @@ const CheckboxItem = styled.li<{ checked: boolean }>`
 
 export const Checkbox = ({
   label,
-  checked,
   onChange,
   singleCheckbox,
 }: CheckboxProps): JSX.Element => {
   return (
-    <CheckboxItem as={singleCheckbox ? 'div' : 'li'} checked={checked}>
+    <CheckboxItem as={singleCheckbox ? 'div' : 'li'}>
```

Comment:
We need to keep checked so that the users selection is retrieved from the context provider, otherwise the check boxes are blank when they return to the page.